bundles/ranvier-combat/behaviors/item/decay.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+module.exports = srcPath => {
+  const ItemType = require(srcPath + 'ItemType');
+  const Logger = require(srcPath + 'Logger');
+
+  let decayStarted,
+      decayEnd = Infinity;
+
+  return {
+    listeners: {
+      updateTick: state => function (config) {
+        let { duration = 60 } = config;
+        duration = duration * 1000;
+        const now = Date.now();
+
+        if (decayStarted) {
+          if (decayEnd < now) {
+            destroyItem(state, this);
+          } else {
+            checkForHalfRotted(this, duration, now);
+          }
+        } else {
+          decayStarted = true;
+          decayEnd = Date.now() + duration;
+        }
+      }
+    }
+  };
+
+  function destroyItem(state, rottedItem) {
+    Logger.verbose(`${rottedItem.id} has decayed.`);
+    state.ItemManager.remove(rottedItem);
+
+    if (rottedItem.room) {
+      rottedItem.room.removeItem(rottedItem);
+    }
+
+    if (rottedItem.type === ItemType.CONTAINER) {
+      rottedItem.inventory.forEach(item => state.ItemManager.removeItem(item));
+    }
+
+    if (rottedItem.container) {
+      rottedItem.container.removeItem(this);
+    }
+  }
+
+  function checkForHalfRotted(item, duration, now) {
```

Comment:
Personally I'd prefer that the `look` command just be updated to check for the `decay` behavior and show the _exact_ time remaining. I have a strong distaste for MUDs that hide numbers when there is no good reason